zip-proportional-output-fee-mechanism-pofm.rst

```diff
@@ -0,0 +1,195 @@
+::
+
+  ZIP: XXX
+  Title: Proportional Output Fee Mechanism (POFM)
+  Owners: Aditya Bharadwaj <nighthawk24@gmail.com>
+  Credits: Madars Virza, Kris Nuttycombe
+  Status: Draft
+  Community Status: Request for comments : https://forum.zcashcommunity.com/t/zip-proportional-output-fee-mechanism-pofm/42808
+  Category: Wallet
+  Created: 2022-08-15
+  License: MIT
+
+
+Terminology
+===========
+
+The key words "MUST", "SHOULD", "SHOULD NOT", "MAY", "RECOMMENDED",
+"OPTIONAL", and "REQUIRED" in this document are to be interpreted as
+described in RFC 2119. [#RFC2119]_
+
+"Z" refers to shielded address.
+"T" refers to transparent address.
+
+"POFM" refers to the proposed change in conventional fee as described in 
+this ZIP.
+
+The term "default transaction fee" in this document is in reference
+to Z to Z, T to Z & Z to T transaction fee paid to miners on the Zcash network
+for the work on including the shielded transaction in a block.
+
+
+Abstract
+========
+
+The goal of this ZIP is to change the conventional transaction fees for
+Shielded transactions and get buy-in from wallet developers, miners & Zcash users.
+
+With an updated transaction fee formula, miners will be compensated fairly for
+including transactions with a high number of outputs, while still allowing low fees 
+for regular shielded transaction use cases.
+
```

Comment:
```suggestion

Once the new fee policy is enforced, this will additionally increase the cost of creating
transactions that collectively have many inputs or outputs, discouraging usage patterns
that cause either intentional or unintentional denial of service.
```

Reply:
Github did not allow adding this change as it was outdated, so I've committed it manually.